tokio-util/src/either.rs

```diff
@@ -9,7 +9,7 @@ use tokio::io::{AsyncBufRead, AsyncRead, AsyncSeek, AsyncWrite, ReadBuf, Result}
 
 /// Combines two different futures, streams, or sinks having the same associated types into a single type.
 ///
-/// This type implements common asynchronous traits provided by `tokio`, and also `std::future::Future`.
+/// This type implements common asynchronous traits such as [`Future`] and those in Tokio.
```

Comment:
You need to also define the link somewhere.

Reply:
Done.